_posts/2019-04-17-cockpit-192.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Cockpit 192
+author: pitti
+date: 2019-04-17
+tags: cockpit linux
+slug: cockpit-192
+category: release
+summary: Cockpit with Machines and translation improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 192.
+
+### Machines: Auto-detect guest operating system
+
+When creating a new VM, the "OS Vendor" and "Operating System" fields are now
+set automatically after specifying the installation sources. This uses the
+libvirt [osinfo-detect](https://libosinfo.org/) utility.
+
+![Machines OS auto-detection](/images/machines-os-autodetect.png)
+
+### Translation cleanup
+
+The language dialog now offers all available translations instead of just a
+fixed subset, orders them alphabetically, and all languages are now shown with
+their native translation:
+
+![New language dialog](/images/language-dialog-translated.png)
+
+Also, translations with less than 50% completion coverage got removed.
+
+### Allow accounts with non-standard shells
+
+Previously, logging into Cockpit was restricted to users who have a shell
+specified in [/etc/shells](https://linux.die.net/man/5/shells), enforced
+through [`pam_shells`](https://linux.die.net/man/8/pam_shells). This was found
+to be too strict (desktop logins don't make this restriction), and excludes
```

Comment:
> (desktop logins don't make this restriction)

Not sure if this is relevant?

Reply:
It's a comparison how Cockpit is more akin to a desktop (GNOME/KDE) login than an ssh login. But I'm happy to drop the parentheses, they are just some more justification/explanation, but maybe too technical.